assets/constructor_macOS_i386.patch

```diff
@@ -1,12 +0,0 @@
-diff -Naur constructor-orig/osxpkg.py constructor-patched/osxpkg.py
---- constructor-orig/osxpkg.py	2022-06-02 23:30:08.000000000 +0200
-+++ constructor-patched/osxpkg-patched.py	2022-06-02 23:31:41.000000000 +0200
-@@ -156,6 +156,8 @@
-         options.set('customize', 'never')
-         options.set('customLocation', '/Applications/MNE-Python')
-         options.set('rootVolumeOnly', 'true')
-+        # Force to run installer through Rosetta 2 on Apple Silicon
-+        options.set('hostArchitectures', 'x86_64')
```

Comment:
including this via conditional. I looked briefly trying to do it myself in `osxpkg.py` but didn't see any existing Intel vs ARM64 conditional so wasn't sure which variable(s) to use :(